static/includes/MigratingVMs.md

```diff
@@ -28,26 +28,20 @@ To migrate VMs from TrueNAS 24.10 (latest) to 25.04:
    - Note the **Zvol path**, storage **Mode**, and **Disk Sector Size**.
    - Document all VM devices and their configurations for later use.
 
-3. Locate or Download the OS Image File
-   - Access the VM via **Display** or **Serial Shell** to confirm the installed OS and version.
-      Refer to documentation for the installed OS, if needed.
-   - Check if the **ISO file** used for installation is stored on TrueNAS or in an external location.
-   - If needed, download a matching OS image before proceeding.
-
 #### Recreating VMs in 25.04
 
 1. Configure Virtualization Settings
-   - After upgrading to TrueNAS 25.04, go to **Instances**.
+   - After upgrading to TrueNAS 25.04, go to **Instances** (formerly **Virtualization**).
    - Click **Select Pool** to open **Global Settings**.
    - Use the **Pool** dropdown to select a storage pool for virtualization.
    - Accept default networking settings or modify as needed, then click **Save**.
 
 2. Recreate the VM
    - Click **Create New Instance**.
    - Select **VM** as the **Virtualization Method**.
-   - Upload or select the ISO file for installation.
-   - Reconfigure the VM using the settings you recorded before updating.
-   - Under **Disks**, select the existing zvol for storage.
+   - Select **Use zvol with previously installed OS** from **VM Image Options**.
+   - Enter or browse to select the path to the existing zvol.
+   - Adjust VM settings (CPU, memory, networking, etc.) to match your previous configuration.
```

Comment:
same as comment above, proper documentation of procedures is to use an ordered not bullet list when steps are performed in a particular sequence.
```suggestion
  a. Select **Use zvol with previously installed OS** from **VM Image Options**.
  b. Enter or browse to select the path to the existing zvol.
  c. Adjust VM settings (CPU, memory, networking, etc.) to match your previous configuration.
```